libs/vscode-ui/components/src/lib/select/select.component.html

```diff
@@ -4,10 +4,8 @@
 
 <div [formGroup]="parentFormGroup">
   <select
-    #selectControl
     (change)="updateValue($event.target.value)"
     role="combobox"
```

Comment:
With Reactive forms, you do not set [value], the control values are set on the FormGroup and connected with the formControl or formControlName directives.